### PR TITLE
Update create_stage_channel to support all features

### PR DESCRIPTION
**Pull request by NarmyOnDiscord**
## Summary

Adds support while creating stage channels for
- bitrate
- user_limit
- rtc_region
- video_quality_mode

Also removes topic, since thats not needed and your code would fail when you try to create a stage channel without a topic.

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [x] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by Rapptz — CHANGES_REQUESTED** [review]
Thanks. These need `versionadded` strings.

File: discord/guild.py [hunk 1]
```diff
@@ -1466,12 +1476,22 @@ async def create_stage_channel(
...
+            options['bitrate'] = bitrate
+
+        if user_limit is not MISSING:
+            options['user_limit'] = user_limit
+
+        if rtc_region is not MISSING:
+            options['rtc_region'] = None if rtc_region is None else rtc_region
+
+        if video_quality_mode is not MISSING:
+            options['video_quality_mode'] = video_quality_mode.value
```
Comment (Rapptz) [inline discord/guild.py:1493]: This also needs a type check like the other methods

**Comment by NarmyOnDiscord** [comment]
> Thanks. These need `versionadded` strings.

2.2, right?

**Review by NarmyOnDiscord — COMMENTED** [review]
Added in the last commit

**Review by Rapptz — APPROVED** [review]
Thanks!

**Pull request merged**